doc/api/deprecations.md

```diff
@@ -2368,6 +2368,38 @@ Type: Runtime
 
 The `_stream_wrap` module is deprecated.
 
+<a id="DEP0126"></a>
+### DEP0126: timers.active()
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/26760
+    description: Runtime deprecation.
+-->
+
+Type: Runtime
+
+The previously undocumented `timers.active()` is deprecated.
+Please use the publicly documented [`timeout.refresh()`][] instead.
+If re-referencing the timeout is necessary, [`timeout.ref()`][] can be used
+with no performance impact since Node 10.
+
+<a id="DEP0127"></a>
+### DEP0127: timers._unrefActive()
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/26760
+    description: Runtime deprecation.
+-->
+
+Type: Runtime
+
+The previously undocumented and "private" `timers.active()` is deprecated.
+Please use the publicly documented [`timeout.refresh()`][] instead.
```

Comment:
Does `timers._unrefActive()` really do the same thing as `timeout.refresh()`?

Reply:
It's an awkward combination of _both_. It is essentially "refresh this timer as unrefed". Active is the same, but refed.

Perhaps there was an extra meaning back when `{un}ref()` made separate libuv handles. I'm not sure but that doesn't exist anymore regardless.